### PR TITLE
[ServiceBus] Change CreateSession to AcceptSession + overloads

### LINKED ISSUE #15579: Use AcceptSessionAsync and AcceptNextSessionAsync instead of CreateSessionReceiverAsync
https://gist.github.com/JoshLove-msft/f0bb933f91e33021e5c0fb5ec61be3f8#proposal-5---accecptsession-on-client

### PR DESCRIPTION
**Pull request by MiYanni**
Fixes: https://github.com/Azure/azure-sdk-for-net/issues/15579

Updated client surface to have AcceptSession overloads that take in session ID. Changed CreateSession to AcceptSession. Made SessionId internal in ServiceBusSessionReceiverOptions.

### CODE REVIEW

**Comment by MiYanni** [comment]
@JoshLove-msft I did notice an inconsistency this causes on the client surface. `CreateSessionProcessor` takes in `SessionIds` in its options. Now that `AcceptSession` does not do it that way, does it make sense to have similar overloads for `CreateSessionProcessor`?

**Comment by JoshLove-msft** [comment]
> @JoshLove-msft I did notice an inconsistency this causes on the client surface. `CreateSessionProcessor` takes in `SessionIds` in its options. Now that `AcceptSession` does not do it that way, does it make sense to have similar overloads for `CreateSessionProcessor`?

I think this is okay. With the SessionReceiver, the user is accepting the session themselves. With the Processor, they are optionally giving us a list of sessions that they want to restrict to. They are not responsible for accepting the session themselves.

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusSessionReceiver.cs [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
 using Azure.Messaging.ServiceBus.Plugins;
-using System.Collections.Generic;
 
 namespace Azure.Messaging.ServiceBus
 {
     /// <summary>
     /// The <see cref="ServiceBusSessionReceiver" /> is responsible for receiving <see cref="ServiceBusReceivedMessage" />
     ///  and settling messages from session-enabled Queues and Subscriptions. It is constructed by calling
-    ///  <see cref="ServiceBusClient.CreateSessionReceiverAsync(string, string, ServiceBusSessionReceiverOptions, CancellationToken)"/>.
+    ///  <see cref="ServiceBusClient.AcceptNextSessionAsync(string,string,Azure.Messaging.ServiceBus.ServiceBusSessionReceiverOptions,System.Threading.CancellationToken)"/>.
```
Comment (JoshLove-msft) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusSessionReceiver.cs:19]: nit: spacing

**Review by JoshLove-msft — APPROVED** [review]

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusSessionReceiver.cs (hunk 1 above)
Comment (MiYanni) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusSessionReceiver.cs]: Fixed.

**Pull request merged**